libcasr/fuzz/init_corpus/js

```diff
@@ -0,0 +1,13 @@
+Error: 1
```

Comment:
Now, you have to update gdb corpus also as numbers changed

Reply:
Done